CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Create a harvest job when a live is ended
 - Component to switch a live to VOD
 
+### Changed
+
+- Medialive profiles use 24FPS instead of 30
+- Increase Medialive profiles segment to 4 seconds
+- Increase Mediapackage endpoint segment to 4 seconds
+- Rollback Medialive control rate mode to CBR
```

Comment:
not sure if it's possible to give some context for these changes. Maybe group them in two or 3 items, give less details but explain why?

Reply:
The explanation is in the commit message, I think it's where it should be. And I choose to describe each modification here because for me they are important, this is critical I think, because I want to found easily in which commit I made this change, because it gives information about how to configure a streaming software like OBS. 
First time in a review someone tell me my changelog is too verbose 😅

Reply:
ok then ^^